spec/rdf2marc/rdf2model/mappers/leader_spec.rb

```diff
@@ -89,30 +88,8 @@
     end
 
     include_examples 'mapper', described_class
-    skip('NEEDS FIX: tests for encoding level and cataloging form are the same')
   end
 
-  describe 'cataloging form' do
-    let(:ttl) do
-      <<~TTL
-        <#{admin_metadata_term}> <http://id.loc.gov/ontologies/bflc/encodingLevel> <http://id.loc.gov/vocabulary/menclvl/f>.
-        <http://id.loc.gov/vocabulary/menclvl/f> <http://www.w3.org/2000/01/rdf-schema#label> "full".
-      TTL
-    end
-
-    let(:model) do
-      {
-        bibliographic_level: 'item',
-        type: 'language_material',
-        encoding_level: 'full'
```

Comment:
I do wonder what cataloging form is and if we never wrote the test right for it ...

Reply:
Cataloging form is the same as description conventions, which is tested below. I fixed the label of this test.

So now everything should be tested and labeled correctly. Thanks for pushing on this.